CONTRIBUTING.md

```diff
@@ -0,0 +1,46 @@
+# Contributing
+
+Thank you for your interest in contributing to this project! Here are some guidelines to get you started.
+
+## Pre-requisites
+
+The easiest way to get started is to use [Docker](https://www.docker.com/) with [Docker Compose](https://docs.docker.com/compose/).
+This project has a `compose.yaml` file which will set up a development environment for you.
+
+## Setup
+
+1. Clone this repository
+2. Clone the [Basic_Framework](https://github.com/SjonHortensius/Basic_Framework) repository
+  - TODO: Simplify this step
```

Comment:
Happy to get any suggestion to simplify this.

It should be possible to pull-it when building the php-fpm image.

Reply:
what can I do to simplify this? Would it help if the framework became a git submodule of the project for example?

Reply:
A git submodule may help yes. I think the best would be to have a composer package on packagist here. But the framework does not seem to follow psr-4.

The submodule is probably the best option at this time. This can be done in a later step.